Dockerfile

```diff
@@ -113,46 +107,19 @@ RUN /opt/eclipse/eclipse -application org.eclipse.equinox.p2.director\
 	-installIUs org.springframework.ide.eclipse.boot.dash\
-	-noSplash
-RUN wget -q https://vorboss.dl.sourceforge.net/project/pydev/pydev/PyDev%207.2.1/PyDev%207.2.1.zip -O /tmp/pydev.zip;\
-    unzip /tmp/pydev.zip -d /opt/eclipse;\
-    rm /tmp/pydev.zip
-RUN wget -q https://projectlombok.org/downloads/lombok.jar -O /usr/local/lib/lombok.jar;\
+	-noSplash && \
+    wget -q https://vorboss.dl.sourceforge.net/project/pydev/pydev/PyDev%207.2.1/PyDev%207.2.1.zip -O /tmp/pydev.zip && \
+      unzip /tmp/pydev.zip -d /opt/eclipse && \
+      rm /tmp/pydev.zip && \
+    wget -q https://projectlombok.org/downloads/lombok.jar -O /usr/local/lib/lombok.jar && \
     java -jar /usr/local/lib/lombok.jar install /opt/eclipse
-COPY rules.java /usr/local/toolchain/rules.java
-COPY rules.python /usr/local/toolchain/rules.python
-COPY README.md /usr/local/toolchain/README.md
-COPY pmd_rules.xml /usr/local/toolchain/pmd_rules.xml
-COPY inproject/shippable.yml /usr/local/toolchain/inproject/shippable.yml
-COPY inproject/tools/testenv /usr/local/toolchain/inproject/testenv
-COPY inproject/functions.local.xslt /usr/local/toolchain/inproject/functions.local.xslt
-COPY inproject/docbook.local.xslt /usr/local/toolchain/inproject/docbook.local.xslt
-COPY etc/m2/settings.xml /usr/local/toolchain/etc/m2/settings.xml
-COPY etc/workbench.xmi /usr/local/toolchain/etc/workbench.xmi
-COPY tools/entrypoint /usr/local/toolchain/tools/entrypoint
-COPY tools/getbranch /usr/local/toolchain/tools/getbranch
-COPY tools/setenv /usr/local/toolchain/tools/setenv
-COPY tools/countTestcases /usr/local/toolchain/tools/countTestcases
-COPY tools/Script /usr/local/toolchain/tools/Script
-COPY tools/code2model /usr/local/toolchain/tools/code2model
-COPY tools/generate_keystore /usr/local/toolchain/tools/generate_keystore
-COPY tools/prepare /usr/local/toolchain/tools/prepare
-COPY tools/environment /usr/local/toolchain/tools/environment
-COPY tools/publish /usr/local/toolchain/tools/publish
-COPY tools/pullanalize /usr/local/toolchain/tools/pullanalize
-COPY tools/getGithubIssues /usr/local/toolchain/tools/getGithubIssues
-COPY tools/orgName /usr/local/toolchain/tools/orgName
-COPY tools/createPypirc /usr/local/toolchain/tools/createPypirc
-COPY tools/makeReport /usr/local/toolchain/tools/makeReport
-COPY tools/pyTestRunner /usr/local/toolchain/tools/pyTestRunner
-COPY tools/pomchecker /usr/local/toolchain/tools/pomchecker
-COPY tools/update-metamodel /usr/local/toolchain/tools/update-metamodel
-COPY tools/pomchecker.xslt /usr/local/toolchain/tools/pomchecker.xslt
-COPY tools/notCreatingDocumentationInPullRequest /usr/local/toolchain/tools/notCreatingDocumentationInPullRequest
-COPY tools/versioncheck.py /usr/local/toolchain/tools/versioncheck.py
-COPY tools/versions.xslt /usr/local/toolchain/tools/versions.xslt
-COPY tools/checkDocErrors /usr/local/toolchain/tools/checkDocErrors
-COPY tools/testBuild /usr/local/toolchain/tools/testBuild
+
+RUN apt-get -y install zenta zenta-tools
```

Comment:
Installation of Zenta and Zenta Tools are in a separate layer as they are updated frequently.